### ISSUE TITLE
Feature Request crash: java.lang.NoSuchFieldException

### ISSUE THREAD

**Issue by nelljohan** [Instabug/Instabug-Android#169]
### Steps to Reproduce the Problem

- Open the Feature Request window using `FeatureRequests.show()`
- Tap either "All Features" or "My Features" at the top

### Expected Behavior

The app to continue functioning as normal

### Actual Behavior

Crash

```
W/System.err: java.lang.NoSuchFieldException: No field mFocusedTextColor in class Lcom/google/android/material/textfield/TextInputLayout;java.lang.Class.getDeclaredField(Native Method)
com.instabug.featuresrequest.c.i.a(TextInputLayoutHelper.java:21)
com.instabug.featuresrequest.ui.e.b.initContentViews(AddNewFeatureFragment.java:245)
com.instabug.featuresrequest.ui.custom.DynamicToolbarFragment.initViews(DynamicToolbarFragment.java:52)
com.instabug.library.core.ui.BaseFragment.onCreateView(BaseFragment.java:38)
androidx.fragment.app.Fragment.performCreateView(Fragment.java:2439)
androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1460)
androidx.fragment.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1784)
androidx.fragment.app.FragmentManagerImpl.moveToState(FragmentManager.java:1852)
androidx.fragment.app.BackStackRecord.executeOps(BackStackRecord.java:802)
androidx.fragment.app.FragmentManagerImpl.executeOps(FragmentManager.java:2625)
androidx.fragment.app.FragmentManagerImpl.executeOpsTogether(FragmentManager.java:2411)
androidx.fragment.app.FragmentManagerImpl.removeRedundantOperationsAndExecute(FragmentManager.java:2366)
androidx.fragment.app.FragmentManagerImpl.execPendingActions(FragmentManager.java:2273)
androidx.fragment.app.FragmentManagerImpl$1.run(FragmentManager.java:733)
android.os.Handler.handleCallback(Handler.java:873)
android.os.Handler.dispatchMessage(Handler.java:99)
android.os.Looper.loop(Looper.java:193)
android.app.ActivityThread.main(ActivityThread.java:6718)
 java.lang.reflect.Method.invoke(Native Method)
com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

This seems related: https://www.reddit.com/r/androiddev/comments/ae4i43/does_android_team_starts_to_abandon_hungarian/

### SDK Version
8.1.1

### Android Version
9.0

### Device Model
Pixel 3 XL

**Comment by mzelzoghbi** [comment]
Hi @nelljohan, Thanks for reporting this. Can you please share with us you build tools version, support libraries version, Gradle build tools & wrapper versions?

**Comment by nelljohan** [comment]
Sure, here you go:

```
com.android.tools.build:gradle:3.4.0-beta05
androidx.appcompat:appcompat:1.0.2
com.google.android.material:material:1.1.0-alpha02
gradle:5.1.1
```

Hope this helps 😄

**Comment by renanferrari** [comment]
Running on the same problem here when tapping one of the tabs, but with a different stack trace:

```
Fatal Exception: java.lang.IncompatibleClassChangeError: Class 'com.instabug.featuresrequest.ui.d.b$3' does not implement interface 'com.google.android.material.tabs.TabLayout$BaseOnTabSelectedListener' in call to 'void com.google.android.material.tabs.TabLayout$BaseOnTabSelectedListener.onTabUnselected(com.google.android.material.tabs.TabLayout$Tab)' (declaration of 'com.google.android.material.tabs.TabLayout$1' appears in base.apk)
       at com.google.android.material.tabs.TabLayout$1.onTabUnselected(TabLayout.java:801)
       at com.google.android.material.tabs.TabLayout.dispatchTabUnselected(TabLayout.java:1763)
       at com.google.android.material.tabs.TabLayout.selectTab(TabLayout.java:1747)
       at com.google.android.material.tabs.TabLayout.selectTab(TabLayout.java:1710)
       at com.google.android.material.tabs.TabLayout$Tab.select(TabLayout.java:2055)
       at com.google.android.material.tabs.TabLayout$TabView.performClick(TabLayout.java:2240)
       at android.view.View.performClickInternal(View.java:6577)
       at android.view.View.access$3100(View.java:781)
       at android.view.View$PerformClick.run(View.java:25912)
       at android.os.Handler.handleCallback(Handler.java:873)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:193)
       at android.app.ActivityThread.main(ActivityThread.java:6912)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:860)
```

Instabug SDK version: 8.1.1
Android Version: 9
Devicel model: Motorola One

```
SDK Build tools version: 28.0.3

Support library versions:
androidx.appcompat:appcompat:1.1.0-alpha02
androidx.constraintlayout:constraintlayout:2.0.0-alpha3
androidx.recyclerview:recyclerview:1.1.0-alpha02
androidx.cardview:cardview:1.0.0
androidx.multidex:multidex:2.0.1
androidx.lifecycle:lifecycle-common-java8:2.1.0-alpha02
androidx.lifecycle:lifecycle-extensions:2.1.0-alpha02
androidx.core:core-ktx:1.1.0-alpha04
androidx.fragment:fragment-ktx:1.0.0
com.google.android.material:material:1.1.0-alpha03

Gradle build tools version: 3.3.2
Gradle wrapper version: 4.10.1
```

I have another app with similar configurations, except for AndroidX, where the issue doesn't happen. I'm guessing it's something related to that?

**Comment by mzelzoghbi** [comment]
Would you be able to try this **SNAPSHOT** and let me know if the issue is fixed for you?

```
implementation 'com.instabug.library:instabug:8.2.0.7-SNAPSHOT'
```
you will need to add these lines to your app **build.gradle** before your dependencies
```
repositories {
    ....
    maven {
        url "https://oss.sonatype.org/content/repositories/snapshots"
    }
}
```
This **SNAPSHOT** is completely safe to use and has no debug code at all.

**Comment by renanferrari** [comment]
@mzelzoghbi That worked for me 👍

**Comment by RogelioRo** [comment]
I just got this crash for

`    implementation 'com.instabug.library:instabug:8.5.0'`

```
FATAL EXCEPTION: main
    Process:
    java.lang.IncompatibleClassChangeError: Class 'com.instabug.library.visualusersteps.d$2' does not implement interface 'com.google.android.material.tabs.TabLayout$BaseOnTabSelectedListener' in call to 'void com.google.android.material.tabs.TabLayout$BaseOnTabSelectedListener.onTabUnselected(com.google.android.material.tabs.TabLayout$Tab)' (declaration of 'com.google.android.material.tabs.TabLayout$1' appears in /data/app/com.teliacompany.zone.dev-7D6GY513ZkYdnjDDS-40kw==/base.apk)
        at com.google.android.material.tabs.TabLayout$1.onTabUnselected(TabLayout.java:826)
        at com.google.android.material.tabs.TabLayout.dispatchTabUnselected(TabLayout.java:1799)
        at com.google.android.material.tabs.TabLayout.selectTab(TabLayout.java:1783)
        at com.google.android.material.tabs.TabLayout.selectTab(TabLayout.java:1746)
        at com.google.android.material.tabs.TabLayout$Tab.select(TabLayout.java:2139)
        at com.google.android.material.tabs.TabLayout$TabView.performClick(TabLayout.java:2327)
        at android.view.View$PerformClick.run(View.java:26083)
        at android.os.Handler.handleCallback(Handler.java:789)
        at android.os.Handler.dispatchMessage(Handler.java:98)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6938)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)

```

**Comment by mzelzoghbi** [comment]
Hey @RogelioRo , this issue has been solved and released in latest SDK version 8.7.0. please try to upgrade and let us know if you faced the same issue again.

**Comment by RogelioRo** [comment]
It is fixed. Thanks!
